src/main/java/pl/wedding/management/model/reservation/Reservation.java

```diff
@@ -0,0 +1,29 @@
+package pl.wedding.management.model.reservation;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private LocalDateTime reservationStart;
```

Comment:
Domyślna konfiguracja zawiera też nanosekundy. Domyślam się, że nie potrzebujemy takiej precyzji, prawda? :)

Reply:
Nie zrobimy naliczania nanosekundowego? Byłoby to innowacyjne podejście ;)

Reply:
A poważnie, dodałem do konwertera obcinanie sekund.